packages/grid_client/src/helpers/types.ts

```diff
@@ -16,9 +16,9 @@ interface ComputeCapacity {
 
 interface BaseMountData {
   /** The name of the mount */
-  name: string;
+  name?: string;
   /** The mount point in the filesystem */
-  mountPoint: string;
+  mountPoint?: string;
 }
 
 interface ExtendedMountData extends BaseMountData {
```

Comment:
Here isnstead of update `BaseMountData` interface we could just extends `Partial<BaseMountData>` which will lead to the same result while avoiding edit any old code that might be used anywhere ... what do u think? @Mahmoud-Emad

Reply:
Fixed, please re-review.